src/IECoreArnold/Renderer.cpp

```diff
@@ -3516,13 +3524,27 @@ class ArnoldGlobals
 				const AtParamEntry *parameter = AiNodeEntryLookUpParameter( AiNodeGetNodeEntry( options ), arnoldName );
 				if( parameter )
 				{
-					const IECore::Data *dataValue = IECore::runTimeCast<const IECore::Data>( value );
+					const IECore::Data *dataValue = nullptr;
+					if( value )
+					{
+						dataValue = IECore::runTimeCast<const IECore::Data>( value );
+					}
+					else
+					{
+						auto it = g_optionDefaultOverrides.find( name );
+						if( it != g_optionDefaultOverrides.end() )
+						{
+							dataValue = it->second.get();
+						}
+					}
+
 					if( dataValue )
 					{
 						ParameterAlgo::setParameter( options, arnoldName, dataValue );
 					}
 					else
 					{
+
```

Comment:
This extra newline looks like it was added by accident?

Reply:
Thanks for spotting that - fixed in https://github.com/GafferHQ/gaffer/pull/5091/commits/b3dccfab8d5947d85ec0bf81ee2dd2d517f327f3.